locksmith/migrations/20220510110232-create-verifier.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Verifiers', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      verifierAddress: {
+        type: Sequelize.STRING
+      },
+      lockAddress: {
+        type: Sequelize.STRING
+      },
+      network: {
+        type: Sequelize.NUMBER
+      },
+      lockManager: {
+        type: Sequelize.STRING
+      },
```

Comment:
Just to double confirm here - default is allowNull. If you don't want anyone to push null on any of the fields there while creating the column, please set to `allowNull` to true.